pkgs/by-name/co/codebook/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  perl,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "codebook";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "blopker";
+    repo = "codebook";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-DDh9yQn5KqsKTi0YP4ulxVMmZv74CgaLA7QF67TnfYY=";
+  };
+
+  buildAndTestSubdir = "crates/codebook-lsp";
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-mrC3fj6BJIpQRHVbMet7J0Vu9qowXJvm6D47YNK5eMg=";
+
+  nativeBuildInputs = [
+    perl
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  # Integration tests require internet access for dictionaries
+  doCheck = false;
+
+  meta = {
+    description = "An unholy spellchecker for code";
```

```suggestion
    description = "Unholy spellchecker for code";
```
See [README.md#meta-attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
> Not start with the definite or an indefinite article.